### ISSUE TITLE
`ccov-clean` not compatible on Windows

### ISSUE THREAD

**Issue by mscofield0** [StableCoder/cmake-scripts#24]
Hi, right now the `ccov-clean` target uses `rm` to remove the `binaries.list` and `profraw.list` files. 

It would be nice not to have to jump in WSL to generate code coverage reports.

**Comment by StableCoder** [comment]
Hi.

I've updated that to be platform agnostic as of [1d4bcb4](https://github.com/StableCoder/cmake-scripts/commit/1d4bcb41000e7a6a0e6ee1350d91f84f03c14320). This should allow builds of individual ccov targets on Windows.

Unfortunately, the ccov-all* targets are still non-functional on Windows, and I'm still trying to figure out a solution.

**Comment by mscofield0** [comment]
Thanks for the quick fix. Closing this now.

**Comment by mscofield0** [comment]
Seems like the Linux version isn't working now:

```bash
$ cmake --build build/test --target ccov
[0/2] Re-checking globbed directories...
[1/13] cd .../build/test/ccov/profraw.list
FAILED: CMakeFiles/ccov-clean
cd .../ModernCppStarterLinux/build/test && /usr/bin/cmake -E rm -f .../build/test/ccov/binaries.list && /usr/bin/cmake -E rm -f .../build/test/ccov/profraw.list
CMake Error: cmake version 3.16.3
Usage: /usr/bin/cmake -E <command> [arguments...]
```

**Comment by StableCoder** [comment]
Fixed via [2a6517a](https://github.com/StableCoder/cmake-scripts/commit/2a6517abb2aa029d4281368a2750165412dd9b40).

In this particular case, `cmake -E rm` was only added to CMake with v3.17.0. I've added a workaround for prior versions that uses the older `cmake -E remove` command instead.
